deebot_client/commands/xml/true_detect.py

```diff
@@ -0,0 +1,21 @@
+"""True detect command module."""
+
+from deebot_client.events import TrueDetectEvent
```

Comment:
Are you sure xml based bot have this command? Even not all json model have it.

Please implement only commands, which are supported by your model and therefore can also be tested.
With the capabilities we can specify which command is supported by which model